Content.Server/Body/Systems/LungSystem.cs

```diff
@@ -50,16 +56,38 @@ private void OnGotEquipped(Entity<BreathToolComponent> ent, ref GotEquippedEvent
 
     private void OnComponentInit(Entity<LungComponent> entity, ref ComponentInit args)
     {
-        var solution = _solutionContainerSystem.EnsureSolution(entity.Owner, entity.Comp.SolutionName);
-        solution.MaxVolume = 100.0f;
-        solution.CanReact = false; // No dexalin lungs
+        if (_solutionContainerSystem.EnsureSolution(entity.Owner, entity.Comp.SolutionName, out var solution))
+        {
+            solution.MaxVolume = 100.0f;
+            solution.CanReact = false; // No dexalin lungs
```

Comment:
```suggestion
            solution.MaxVolume = entity.Comp.MaxVolume;
            solution.CanReact = entity.Comp.CanReact;
```
Fuck it, if we're touching this system, I might as well ask you to make these into DataFields on the LungComponent. A long, long time ago I wanted Harpies to have 2x lung volume, and then got mad because "Some system somewhere" was hardcoding the lung volume to only ever be 100.